devdox/app/schemas/repo.py

```
@@ -83,59 +84,112 @@ class GitRepoResponse(BaseModel):
+            "html_url": repository.html_url,
+            "private": repository.private,
+            "visibility": getattr(repository, "visibility", None),
+            "repo_created_at": repository.created_at,
+        }
+
+    @classmethod
+    def from_github(cls, data: Repository | SimpleNamespace | dict) -> GitRepoResponse | None:
+
+        if not data:
+            return None
+        elif isinstance(data, Repository) or isinstance(data, SimpleNamespace):
```

Ruff (0.11.9)

Multiple `isinstance` calls for `data`, merge into a single call

Merge `isinstance` calls for `data`

(SIM101)